source/extensions/tracers/xray/util.cc

```diff
@@ -0,0 +1,50 @@
+#include "extensions/tracers/xray/util.h"
+
+#include "absl/strings/ascii.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace XRay {
+
+bool WildcardMatch(absl::string_view pattern, absl::string_view input) {
```

Comment:
wildcardMatch().

This code looks like it wants to be fuzzed.

Reply:
I'll rename it. It will take me sometime to get used to the google style :/

Reply:
Actually that *is* Google style for functions. Envoy style for methods is lowerFirst().

Reply:
both are terrible 😄 they're are very Java/C#-ish IMHO.
The C and the C++ standards follow snake_case so I don't know why people don't just follow that. oh well.